lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -953,6 +961,12 @@ def fake_tarfile_open(self, path):
                 tb.filestream("subdir1/f11.txt")
             assert str(exc.value) == expected_error_msg
 
+            path = Path(tb.name)
+            expected_error_msg = f"An error occurred while unpacking {tb.tarball_path}: Unable to extract {str(path)!r}"
+            with pytest.raises(TarballUnpackError) as exc:
+                tb.filestream("")
```

Comment:
If the request is `GET /api/v1/datasets/inventory/{dataset}/`, will the `target` value be `""` or `None`?  (I _think_ it will be `None`, but it's not a bad idea to be prepared for and test both...but I think they should both produce the same result.)

Reply:
Just FYI, that's `GET /api/v1/datasets/{dataset}/inventory/` 😄 And now I'm trying to remember what Flask will give us with a missing URI parameter, and I'm not entirely sure. 🤔

Reply:
> that's GET /api/v1/datasets/{dataset}/inventory/ 😄

Ooops...I copied it from [our source](that's GET /api/v1/datasets/{dataset}/inventory/ 😄)....  🤣 

> I'm trying to remember what Flask will give us with a missing URI parameter

[Our code](https://github.com/distributed-system-analysis/pbench/blob/4c08fd86ccececa9a195e302fafe602d4a0ae4d0/lib/pbench/server/api/resources/datasets_inventory.py#L80) thinks it will be `None`....  🙃

Reply:
Now, I am also unsure what empty value of target is interpreted as